docs/knowledgebase/runtime/pallets.md

```diff
@@ -60,15 +65,68 @@ decl_storage! { ... }
 // actions this pallet takes throughout block execution.
 decl_module! { ... }
 ```
+> **Note**: Additional sections such as [`decl_error!`](./macros#decl_error) are commonly used however are not a requirement for a minimally working pallet.
+#### Example
+
+The [Basic Token recipe](https://substrate.dev/recipes/basic-token.html) provides a good example of a simple FRAME V1 pallet that allows a user to create a `u64` supply of tokens in storage and a `StorageMap` of account IDs and `u64` balances. The primary parts include:
+- `decl_storage!` defines `TotalSupply` and `Balances` the pallet's storage items;
+- `decl_event!` defines `Initialized` and `Transfer` events;
+- `decl_module!` defines the pallet's functions, `init()` and `transfer()`; and
+- `decl_module!` also has a `deposit_event()` function, common to all pallets that define behavior that emits an event.
+### FRAME v2
+A FRAME v2 pallet is commonly composed of 7 sections:
+
+```rust
+// 1. Imports and Dependencies
+pub use pallet::*;
+#[frame_support::pallet]
+pub mod pallet {
+	use frame_support::{dispatch::DispatchResultWithPostInfo, pallet_prelude::*};
```

Comment:
DispatchResultWithPostInfo should be in the pallet_prelude already, is there a reason you reimport ?